test/test_read_preferences.py

```diff
@@ -504,7 +504,7 @@ def test_read_preference_document_hedge(self):
             with self.assertRaises(TypeError):
                 cls(hedge=[])  # type: ignore
 
-            pref = cls(hedge={})
+            pref = cls(hedge={})  # type: ignore[abstract]
```

Comment:
What's the error here?

Reply:
was there because _ServerMode's `__call__` was abstract. After talking irl, its now not abstract and these type ignore's have been removed